### DISCUSSION TITLE
Matrix channel

### DISCUSSION THREAD

**Discussion by Gerschtli** [nix-community/nix-on-droid#220] [General]
Hey, I created a matrix channel, maybe it could help a bit with communication when having a chat. Decided to use matrix because nix community already uses it. Would be great to link it in the repo description.

https://matrix.to/#/#nix-on-droid:matrix.org

**Comment by t184256** [comment]
I'm not really a fan of Matrix (the protocol) and haven't really used it, but I'd be interested in hanging around this particular channel. Do you know, by chance, whether it is something I can use from my within my favourite Jabber/XMPP client/account? I'm willing to install some bridge for it, esp. if it's packaged in NixOS.

**Reply by Gerschtli** [reply]
I am really new to matrix, so I can not give any recommendations. Just thought it would be good to use the same tool as the rest of the community is already using
